src/plugins/indicator_libmessaging.py

```diff
@@ -41,6 +47,10 @@ def activate(self, app, source):
     # show the number of unread messages and subscriptions
     # on the messaging menu
     def show_unread(self, draw_attention=False):
+        """
+        show the number of unread messages and subscriptions
+        on the messaging menu
+        """
```

Comment:
So remove the comment if you copied it to docstring.

Reply:
I agree.

Reply:
Okay